website/docs/examples/track_planner/README.md

```diff
@@ -0,0 +1,107 @@
+---
+id: track-planner
+title: Track Planner
+---
+
+:::info Basic Knowledge Requirements
+
+Before diving into this code, here's a quick heads-up on what you'll need to be familiar with:
+
+1. **Python Programming**: It's important to have a good grasp of Python, especially with concepts
+like `functions`, `loops`, and `classes`, since the example utilizes these fundamentals.
+2. **Asynchronous Programming with asyncio**: Familiarity with Python's asyncio for writing concurrent
+code using the `async/await` syntax.
+3. **[farm-ng Filter Service Overview](/docs/concepts/filter_service/)**:
+This overview provides a base understanding of the gRPC service the client you create will connect to.
```

Comment:
The filter service page makes no explicit mention of the gRPC

Reply:
I think **[Amiga Development 101]**(https://amiga.farm-ng.com/docs/concepts/system_overview/) would be the place to make it clear that services communicate over and are connected to via gRPC. Maybe we make it explicit in most/all of the examples that going over **[Amiga Development 101]**(https://amiga.farm-ng.com/docs/concepts/system_overview/) is one of the common prereqs (this would be a separate PR)